requirements_dev.txt

```diff
@@ -10,7 +10,7 @@ numba==0.55.1
 numpy==1.21.5
 openpyxl==3.0.9
 pandas==1.4.1
-rich==11.2.0
+rich==12.0.1
```

Comment:
This is already part of main.

Reply:
I might have rebased it onto the wrong branch 😖 
Resolved after rebasing on the [current main](https://github.com/glotaran/pyglotaran/commit/c8b993b753a4fe9cba64a1c5bd0fb461f3dac05c).